types/external/lang-detector.d.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+declare module 'lang-detector';
```

Comment:
Are there any types we need to define here for use when the components are eventually converted to TypeScript?

Reply:
I was building a proof of concept, so I didn't defined the types.

Reply:
Done